frontend/src/metabase/components/Tooltip.jsx

```diff
@@ -144,6 +117,22 @@ export default class Tooltip extends Component {
   };
 
   render() {
-    return React.Children.only(this.props.children);
+    const { isEnabled, tooltip } = this.props;
+    const isOpen =
+      this.props.isOpen != null ? this.props.isOpen : this.state.isOpen;
+    return (
+      <React.Fragment>
+        {React.Children.only(this.props.children)}
+        {tooltip && isEnabled && isOpen && (
+          <TooltipPopover
+            isOpen={true}
+            target={this}
+            hasArrow
+            {...this.props}
+            children={this.props.tooltip}
```

Comment:
Is there any particular reason why we use `children` as a prop instead of `<TooltipPopover ...>{tooltip}</TooltipPopover>` I'm assuming it's functionally equivalent just a matter of style?

Reply:
Not sure why it was done this way but it is functionally equivalent to `<...>{this.props.tooltip</...>`. If you want it changed I can change it.

Reply:
Nah it's ok as is I think, just caught my eye.